### DISCUSSION TITLE
[i118n] - region/language - second param language?

### DISCUSSION THREAD

**Discussion by JamesBliss** [vercel/next.js#19415] [Help]
Hey,

I want to use the power for Nextjs 10 to handle language-based routing expect my company wants to have `www.example.com/region/language` e.g. `gb/en` or `us/en`.

Is this possible to handle this directly within next? I think I could use a custom server to handle the `region` part but I would prefer to handle this all directly within next?

Thanks in advanced for any ideas or help!

**Comment by weyert** [comment]
It's supporting passing `en-GB`, `en-US` etc. but I don't think you can customise the way it's reflected in the uri
